### ISSUE TITLE
Follow renames on File History cannot load more history

### ISSUE THREAD

**Issue by cateyes99** [eamodio/vscode-gitlens#1148]
- GitLens Version: _10.2.3_
  
- VSCode Version: _1.50.1_
- OS Version: _Windows 10 Enterprise_

Steps to Reproduce:

1. I have 2 files which have been renamed a few times. Both have this issue.
2. With the setting `Follow Renames` on, I found in the `File History` view. after clicking `... Show More Commits`, let it load more history. It works once. See the below screenshot. There is more history in the repo, after the commit `Merged PR 8895: ...` at the bottom (I'm using **Azure DevOps provided repo**, and can see them in Azure DevOps repo). 
![image](https://user-images.githubusercontent.com/19678455/98919545-d4484d00-2533-11eb-9850-e23fa5e4d00f.png)
3. Then clicking `... Show More Commits` again, GitLens doesn't load further history, and the **clickable** `... Show More Commits` **disappears**.
![image](https://user-images.githubusercontent.com/19678455/98919952-618ba180-2534-11eb-8546-29e252112671.png)
**NOTE**: When it reaches the commit `Merged PR 8895: ...`, it has already passed one file rename, i.e., that commit is right in the middle of 2 file renames. Not sure if this piece of info is useful.

**Comment by cateyes99** [comment]
@eamodio i just checked out `Gitlens Version 11.0.5` with `VSCode Version: 1.51.1`, found that for the previous file I mentioned in my **OP**, the new version doesn't even show up the `... Show More Commits` button in the `File History` view with `Toggle Renames: Following`. Further more, it shows one commit less than `Toggle Renames: Not Following` See below:
- `Toggle Renames: Not Following`
![image](https://user-images.githubusercontent.com/19678455/100494157-d7824080-31a3-11eb-9ce9-1e4d79c9cd1d.png)

- `Toggle Renames: Following`
![image](https://user-images.githubusercontent.com/19678455/100494186-2a5bf800-31a4-11eb-9142-f360ad8fe545.png)
We can see the last commit shown in the above screenshot is missing here, and there is no `... Show More Commits` at all 😱

**Comment by eamodio** [comment]
@cateyes99 Was the file renamed in either of those 2 commits?

**Comment by cateyes99** [comment]
Yes. Its 1st rename is in the commit of `Updated the use of Offering Client and tidy ups` marked with the red arrow in the screenshot from [my yesterday's comment](https://github.com/eamodio/vscode-gitlens/issues/1148#issuecomment-735041105).

See, in my [**OP**](https://github.com/eamodio/vscode-gitlens/issues/1148#issue-741434237) the old version's `Follow Renames` shows more history behind the red arrow marked one. But there are more it cannot find.

If i run the command `git log --all --oneline --follow infrastructure\ia-ansprmgr-fn.json` it shows much more history including a few times of rename as well below (Note the last rename mark I put in it). _well it onl fails the very 1st rename and doesn't show the history beyond the 1st rename though, because apart from the rename there were too many changes in content as well, so git doesn't treat it as a rename I think._. If GitLens has used a command similar to that one, it should be able to give all the commits git can find.

```
e4a607b Replace the custom ... service bus trigger with the built-in one & use Managed Identity to access service bus      
979a624 Updated NFTA configuration
663e747 Updated packages to 65
f4dd32e Merged PR 17142: Handle large message payload; Refactor to support EventDocument as message body instead of JObject
1472eb2 Finalized
820d973 Merged PR 16502: Replaced IEnrichedLogger with ILogger
29a7a2d (origin/working/rh_tx) WIP
12838fb WIP
9fad939 WIP
13e8df5 updates for tokenexchange migration phase 1
26ecb4c Merged PR 15700: Update app insights
cacf84f Merged PR 15059: Add VNets to multiregion environments
47a6381 Merged PR 14839: Upgraded to .net core 3.1
ec3468a Merged PR 14451: Add VNet to configuration file
1e91f4a Merged PR 14055: Add FNTE to configuration files
db11d6a Merged PR 13899: Add BT client
6309013 Merged PR 13046: -moving onto version 1.0 from v1 for taks.read fn
50f4e6f Merged PR 12854: Updated IA.API.Write version
22940b3 Merged PR 12799: Code formatting
0e21fbf Merged PR 12759: - changing IA Read version to 1.0
025e97d Merged PR 12400: Updated ia-ansprmgr-fn.json
b695424 Updated the use of Offering Client and tidy ups                                  <== the last rename !!!!!!
314096e Merged PR 11918: Updated ia-answersprocessmanager-fn.json
59ea272 Remove Region 2 From Automation Tests
1ec679e pushing to test in DEXP now
a9d9f78 Updated ia-answersprocessmanager-fn.json
9382a3b Updated ia-answersprocessmanager-fn.json
c3be1f1 Merged PR 10789: Devex Refactor
7ac5d59 Updated app service to use shared service plan and storage account
5534f30 Merged PR 10376: Port to V2
408f87a Merged PR 9719: Remove slot
3ef1ad7 Merged PR 9695: Update to single asp
80a54f7 Merged PR 9099: remove keys from config
b254ce0 Merged PR 8992: [inf] update DINT and DEXP to app service plan
b665144 Merged PR 8895: Update IA.AnswersProcessManager config & build files
ea00705 Merged PR 8750: Switch lower environment to SMOKE tests
b37f4de Tidy ups
a393a55 Temporarily enabling smoke tests in DINT
ca0b4fe [inf] increase lock time for DINT and DEXP
5312cec Merged PR 8187: Update AppServices deploy config
7047f36 Merged PR 8137: Add KeyVault access for NFTA and PROD
e7cce55 Merged PR 8030: removed unused key
8c78731 correcting name of Key Vault
454cc5e Merged PR 7744: Updated ia-answersprocessmanager-fn.json
b782d00 Merged PR 7733: Updated ia-answersprocessmanager-fn.json
01a58f6 Merged PR 7721: Updated ia-answersprocessmanager-fn.json
c285d4f Merged PR 6893: - renamed peek_lock to max_process_time and set to 9 instead of 59
118f4ec Merged PR 6839: Updated ia-answersprocessmanager-fn.json
af87bd8 Replace ip_whitelist block with ip restrictions line
0f61907 Merged PR 6772: Updated ia-answersprocessmanager-fn.json
4554321 Merged PR 6761: fixing apim
0a23b90 Merged PR 6462: - renamed pipeline files
94b08b9 [inf] update to match naming conventions
55e47cb Merged PR 6443: Add function key back to make debugging easier. Later need to remove
43f8708 [src] add missing app settings
f2960de Merged PR 6426: Fixed APIM Endpoint Name
e3f72b1 Update user to match what Token Exchange creates
e5fe6ed Merged PR 6093: Corrected Product Name in Config
662ee15 Fixed config variable name
d6ccaaa Chagned undefined variable
f1e66bb Changed sub id
2df14b6 Changed api-version format
ee7c02b Improved config naming and updated sub id value
84cd37e Updated ia-apm.json
586b5ac Updated ia-apm.json
41856d3 Updated ia-apm.json
41f4ae8 Added keyvault settings
416da89 Updated ia-apm.json
69a79ea [e2etest] Modified config for SpecFlow tests in the pipeline
c536d9f Updated ia-apm.json
9a5e3b6 config changes
ea634f4 config changes
4958e4f apim changes
85bbb42 Merged PR 5727: Fixed Config
01dbe33 Merged PR 5680: Removed Region from Service Bus Deploy
2c107e1 Merged PR 5625: use new if match implementation
b2772a1 Merged PR 5472: Added NFTA/PROD deployments
6b72793 1. reapplied the setting, 2. refactoring the exceptions handling of subscriptionkey provider.
84c2227 revert NFTA / PROD
de7469e revert NFTA / PROD
d5d16ad Add NFTA & PROD configs
3f6c959 Update AppInsights name
2a536c5 Refactor APM config & add pre-reqs for NFTA & PROD
7c7a11b correct the subscriptionid
a9a43ef update config to 1. use api management url 2, change product name
b843cdc missing config
c45fa78 adding  log_analytics_id
2007cf4 adding key_vault_sp
e647caf 1. setup function deployment config to deploy keyvault and seed key 2. changed code to retreive key from keyvault 3. added API version in downstream 
call and add subscriptionKey in the header. 4.Updated NuGet packages.
7a28fa2 Merged PR 5056: lots of little fixes
cd856d0 Merged PR 4981: [inf] Fnta to s2 plan
c58f096 Renamed the digital subscription name and added bounded context tag
98caeda one more line removed
ac7b8d2 v2 changes does not require these fields
f893f51 v2 template
c701f4e service bus key reader role added
50ea8a3 going back to how it was before time
4c9519b as discussed again. hopefully this is the last one.
9de1a01 Merged PR 4613: as discussed
70ed56d now using the connection string
69b2e5a as discussed
6d182f6 i need real values there
dd776d2 arghhh... dint and dexp is dev subsription
417b9ea forgot bracket
e3e0a33 infrastructure changes (revert and change) and updated the local setting template
e571c40 check in infrastructure changes
3b38b9c [inf]Update subscription for dint and dexp
40f3604 [inf]add DINT and DEXP environments
a715649 Merged PR 4278: key corrected to point to fnte
f18b1e3 Assign CosmosDB Master Key Reader role
c0cb989 Merged PR 4074: Updated ia-apm.json to switch to App Service Plan S1
ec7b181 bug resolved. correct variable name.
4752df2 removed connections settings from app.config
f638ee4 final check in for need analysis and cover select
cdd6787 correct setting this time
29a625b refer products correct endpoint
b7feaea Revert "doing this after change in pipeline manager"
d0cff4e doing this after change in pipeline manager
66e9aed devops have done some changes in the pipeline manager and because this does not return a key. it fails because it cant take a null
9b2c56f new unit tests added. and one bug from merges resolved
b728eaa Merged PR 3366: [app] added classes for validate offering request / response and for refer
c106acd Merged PR 3339: Adding Cover Select Section
6e5a159 Merged PR 3332: increase the peek lock duration to 59 as the lock duration on subscription is
3faf999 increase the peek lock duration to 59 as the lock duration on subscription is set as 1 minute
bcd404c task read used now. was incorrectly using task write.
7a61960 bug found. env. code missing.
fd75b64 tags for tests
5aee7b9 config variable names changed
0e50fde updated to if match key
6cd9dab Merged PR 3124: [app] cleanups and added missing appsettings
ba3aa2e Deactivate offline slot after deployment
4a78c2d Add FNTA environment
615444d Update ref endpoints
d5d9622 Fix Service Bus namespace
68bf5a3 Update configuration
c2c7544 Update configuration
9b6a9ee Update configuration
cc99cf6 Update configuration
9d31d9c Update ServiceBus config
7cbd581 Add service bus support
a69115e Update to FNTE env
08df390 Update configuration
3a27168 Update api_code
1f3b060 Update configuration
9919788 Update configuration
dcb37be Update config
aef8b22 Update configuration
```

**Comment by eamodio** [comment]
@cateyes99 opened a new issue for this investigation - #1221
